lib/aptible/cli/subcommands/services.rb

```diff
@@ -44,6 +44,133 @@ def services
 
               service.update!(**updates) if updates.any?
             end
+
+            desc 'services:sizing_policy SERVICE '\
```

Comment:
Do we have a way to retrieve the current policy?

Reply:
whoops, you're right. added